### ISSUE TITLE
In Group Management /Domains width of column 'domain/regex'  is to large with long domains/regex

### ISSUE THREAD

**Issue by yubiuser** [pi-hole/web#1458]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/AdminLTE/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/AdminLTE/issues)).

**How familiar are you with the the source code relevant to this issue?:**

1

---
**Expected behaviour:**

Long domain/regex should be broken to fully display other columns

**Actual behaviour:**

Line breaking is not optimal - column type is not displayed total.

![83690289-c12e6a00-a5f0-11ea-9f18-c41cf9f3c621](https://user-images.githubusercontent.com/26622301/84485809-cc426380-ac9c-11ea-9284-c61c49ced3de.png)

**Steps to reproduce:**

 AdminLTE version is devel v5.0-387-g557bd858 (Latest: v5.0)
Browser: Linux Mint, 19.3, Chrome 83, Firefox 77.

Regex: 

```
(^|[-_.]+)(m?a(d((vert(s|is(ing|e?ments?))?|im(age|g)s?)|([vx]|s(e?rv(er?|ice)?s?)?)|track(ers?|ing))?|ff(iliat(es?|ion))?|nalytics?)|b((anner|eacon)s?)|count(ers?)?|pixels?|stat(s|istics?)?|t(elemetry|ra(ffic|ck(ers?|ing))))[-_]*[0-9]*[-_.]
```

**Comment by yubiuser** [comment]
Still an issue

**Comment by rdwebdesign** [comment]
Fixed by #2078
